src/interface/src/app/map/map-manager.ts

```diff
@@ -467,7 +467,7 @@ export class MapManager {
   private setUpPanHandler(map: L.Map) {
     if (!this.mapViewOptions$.getValue().center) return;
 
-    map.panTo(this.mapViewOptions$.getValue().center);
```

Comment:
can you add a comment on why we're disabling (but leaving) this here?

Reply:
Added!